app.yaml

```diff
@@ -40,9 +40,17 @@ handlers:
     # WARNING TO DEVELOPERS: Files in this folder may be stale for
     # up to 10 mins after a code release.
     Cache-Control: 'public, max-age=600'
+- url: /assets/(constants\.json)
```

Comment:
I think it's actually fine for them to be in 'assets', so that they follow the same procedure as all our other stuff (see scripts/build.py for details). It's probably easier than declaring a new directory, and I think it's fine to think of assets as "statically-retrievable things".